lib/azure/resource_management/ARM_interface.rb

```diff
@@ -29,17 +29,17 @@ class ARMInterface < AzureInterface
       include Azure::ARM::ARMDeploymentTemplate
       include Azure::ARM::VnetConfig
 
-      include Azure::ARM::Resources
-      include Azure::ARM::Resources::Models
+      include Azure::Resources::Mgmt::V2018_05_01
+      include Azure::Resources::Mgmt::V2018_05_01::Models
 
-      include Azure::ARM::Compute
-      include Azure::ARM::Compute::Models
+      include Azure::Compute::Mgmt::V2018_06_01
+      include Azure::Compute::Mgmt::V2018_06_01::Models
 
-      include Azure::ARM::Storage
-      include Azure::ARM::Storage::Models
+      include Azure::Storage::Mgmt::V2018_07_01
+      include Azure::Storage::Mgmt::V2018_07_01::Models

```

Comment:
https://github.com/Azure/azure-sdk-for-ruby/commit/5effef547df2362daf8a1a12021de37598d63e2a

https://rubygems.org/gems/azure_mgmt_storage/versions/0.17.2